plugin/trino-openlineage/src/main/java/io/trino/plugin/openlineage/OpenLineageListenerConfig.java

```diff
@@ -105,4 +109,31 @@ public OpenLineageListenerConfig setNamespace(String namespace)
         this.namespace = Optional.ofNullable(namespace);
         return this;
     }
+
+    public String getJobNameFormat()
+    {
+        return jobNameFormat.orElse(DEFAULT_JOB_NAME_FORMAT);
+    }
+
+    @Config("openlineage-event-listener.job.nameFormat")
+    @ConfigDescription("Set format for job name facet.")
```

Comment:
Remove the following dot.

Reply:
Fixed, including all other ConfigDescription in this file

Reply:
Ok